x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/siem_winlogbeat_ecs/ml/datafeed_windows_anomalous_network_activity.json

```diff
@@ -0,0 +1,24 @@
+{
+    "job_id": "JOB_ID",
+    "indices": [
+      "INDEX_PATTERN_NAME"
+    ],
+    "query": {
+        "bool": {
+          "filter": [
+            {"term": {"event.action": "Network connection detected (rule: NetworkConnect)"}}
+          ],
+          "must_not": [
+            {
+              "bool": {
+                "should": [
+                  {"term": {"destination.ip": "127.0.0.1"}},
+                  {"term": {"destination.ip": "127.0.0.53"}},
+                  {"match": {"destination.ip": "::1"}}
```

Comment:
I don't want to hold this up any longer, but there is still this `match`.

Reply:
Changed in b38031e91e8ca413cf9fa513c4a662eb3fa58222
@randomuserid can you confirm leaving the `match` in wasn't intentional?